closed/OpenJ9.gmk

```diff
@@ -304,7 +304,8 @@ $(foreach file, \
 		testlibA \
 		testlibB \
 		vmruntimestateagent29 \
-		), \
+		) \
+	$(patsubst %, $(OPENJ9_VM_BUILD_DIR)/lib%.jnilib, loadLibraryTest), \
```

Comment:
A minor formatting nitpick - can you make this match the pattern used by the other examples?

```
$(patsubst %, $(OPENJ9_VM_BUILD_DIR)/lib%.jnilib,
	loadLibraryTest \
	) \
```

Reply:
will do